js/views/modals/listingDetail/Listing.js

```diff
@@ -56,27 +82,100 @@ export default class extends BaseModal {
     this.listenTo(this.editModal, 'click-return', onEditModalClickReturn);
   }
 
-  get $btnRetry() {
-    return this._$btnRetry || this.$('.js-retryFetch');
+  onClickDeleteListing() {
+    if (this.destroyRequest && this.destroyRequest.state === 'pending') return;
+
+    this.destroyRequest = this.model.destroy({ wait: true });
+
+    if (this.destroyRequest) {
+      this.$deleteListing.addClass('processing');
+
+      this.destroyRequest.done(() => {
+        if (this.destroyRequest.statusText === 'abort' ||
+          this.isRemoved()) return;
+
+        this.close();
+      }).always(() => {
+        if (!this.isRemoved()) {
+          this.$deleteListing.removeClass('processing');
+        }
+      });
+    }
+  }
+
+  showDataChangedMessage() {
+    if (this.dataChangePopIn || (this.dataChangePopIn && this.dataChangePopIn.isRemoved())) {
```

Comment:
If this.dataChangePopIn is true, the second condition isn't checked, but if it's false, then the second condition will also be false, won't it?